bigquery/bqml/requirements.txt

```
@@ -3,6 +3,7 @@ google-cloud-bigquery-storage==2.27.0
 pandas==1.3.5; python_version == '3.7'
 pandas==2.0.3; python_version == '3.8'
 pandas==2.2.2; python_version > '3.8'
```

Pandas 2.2.**3** comes precombined for Python 3.13
```suggestion
pandas==2.2.3; python_version > '3.8'
```